applications/vessel-history/src/components/Splash.tsx

```diff
@@ -0,0 +1,21 @@
+import React, { memo } from 'react'
```

Comment:
I don't think we need to always render a Splash component with a certain timeout, why not include loading spinners on the UI when needed ?

Reply:
You are right we can just remove the timeout, we only introduced it so that the splash screen is noticeable in faster internet connection speeds.

Reply:
@j8seangel @rdgfuentes I think that the ticket require that the splash show for a moment (maybe to see the icon of sponsor), or I am wrong?